common/utils.go

```diff
@@ -54,6 +58,7 @@ func NewCanal(config SourceConfig) (*canal.Canal, error) {
 	cfg.Password = config.Password
 
 	cfg.IncludeTableRegex = config.Tables
+	cfg.Logger = nopLogger{}
```

Comment:
It was an okeyish amount of logging, but annoying when debugging. But I agree with you that this could be more flexible than just disabling the entire thing, I'll see if I can just disable it for tests, and let it enabled by default.

Also, the log format was different from the loggers conduit uses, so I might actually need to add a custom zerolog logger for canal logs.